src/engine/input/key-mapping/InputEventItem.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using Godot;
+
+public class InputEventItem : Node
+{
+    [Export]
+    public NodePath ButtonPath;
+
+    internal InputActionItem AssociatedAction;
+
+    internal bool JustAdded;
```

Comment:
Why are these internal? If these need to be externally set from somewhere else these should be properties